coops-ui/package.json

```diff
@@ -17,14 +17,11 @@
     "country-list": "^2.2.0",
     "http-status-codes": "^1.3.2",
     "ld-vue": "^0.1.1",
-    "lodash.isempty": "^4.4.0",
-    "lodash.isequal": "^4.4.0",
-    "lodash.omit": "^4.4.0",
+    "lodash": "^4.17.15",
```

Comment:
This is great 👍

Reply:
FYI - importing just the functions you need is _usually_ better, but in this case Vuetify (and other packages) load the whole lodash package anyway so not only are we not smaller but we're actually adding extra packages! :/